test/PosDef.jl

```diff
@@ -0,0 +1,28 @@
+module PosDef
```

Comment:
Is it necessary to put it in a separate module? I would be surprised at least if the use of Requires and separate modules are needed to define the adjoint for Tracker.

I would have assumed that something like
```julia
if AD == "All" || AD == "Tracker"
    include("tracker_utils.jl")
end
```
in test/ad/distributions.jl would be sufficient.

Reply:
The problem is that the `TrackedMatrix` overload is compiled even if `Tracker` is not loaded...
I tried it with Requires in the `test/ad/distributions.jl` file which didn't work, but defining `is_posdef` in `test/runtests.jl` and requiring the Tracker stuff there does the job. So there is no more separate module, but there might be an obvious solution I am not seeing?